packages/salesforcedx-utils-vscode/src/telemetry/telemetry.ts

```diff
@@ -101,11 +101,8 @@ export class TelemetryService {
     const isDevMode = machineId === 'someValue.machineId';
 
     // TelemetryReporter is not initialized if user has disabled telemetry setting.
-    if (
-      this.reporter === undefined &&
-      this.isTelemetryEnabled() &&
-      !isDevMode
-    ) {
+    const isTelemetryEnabled = await this.isTelemetryEnabled();
+    if (this.reporter === undefined && isTelemetryEnabled && !isDevMode) {
```

Comment:
oh...interesting. So what change here that might have been the root cause of the test timeouts is by pulling the assignment to the variable outside of the conditional we now await it every time this is called...where as previously if reported is defined it would never be called after the first attempted unless it was disabled. Might be worth moving back to 
`this.reporter === undefined && await this.isTelemetryEnabled() && !isDevMode` to avoid making the async call everytime

Reply:
Yep, def should switch this back with that in mind.  Updated!